lib/cluster_holding.rb

```diff
@@ -76,4 +71,27 @@ def self.delete_old_holdings(org, date)
     end
   end
 
+  private
+
+  def update_holding(old, new)
+    old.update_attributes(date_received: new.date_received,
+                          uuid: new.uuid)
+  end
+
+  def find_old_holdings(cluster, holding)
+    if cluster.large?
```

Comment:
The parallel conditionals here might suggest that there should be a subclass that knows how to deal with large clusters, but I'm OK with this for now. If we start adding significantly more support around large clusters, we should perhaps rethink it.